source/plugins/followup/README.md

```diff
@@ -21,6 +21,6 @@ The *followup* plugin displays the ratio of open/closed issues and the ratio of
   with:
     # ... other options
     plugin_followup: yes
-    plugin_followup_sections: repositories, user #
+    plugin_followup_sections: repositories, user
```

Comment:
It's probably missing the comment to explain option. It should be completed with a short description instead. (like "Display both issues/pr opened on user's repositories and opened by user" or something like that